{{cookiecutter.repo_name}}/{{cookiecutter.project_name}}/app.py

```diff
@@ -17,16 +19,21 @@ def create_app(debug=False, testing=False, model_only=False):
     Create the object graph for the application.
 
     """
-    loader = load_each(
+    config_loader = load_each(
         load_default_config,
         load_from_environ,
+        load_from_json_file,
+    )
+    partitioned_loader = load_config_and_secrets(
+        config=config_loader,
+        secrets=load_from_dynamodb(),
```

Comment:
With regards to open-sourceness, I'm not sure if this violates that requirement. This loader should do nothing by default.